doc/man1/openssl-cms.pod.in

```diff
@@ -97,11 +98,11 @@ B<openssl> B<cms>
 [B<-inkey> I<file>]
 [B<-keyopt> I<name>:I<parameter>]
 [B<-passin> I<arg>]
-[B<-rand> I<files>]
-[B<-writerand> I<file>]
 [B<-to> I<addr>]
 [B<-from> I<addr>]
 [B<-subject> I<subj>]
+{- $OpenSSL::safe::opt_r_synopsis -}
```

Comment:
Nit: You have different order - trust and then r items down there.

Reply:
good spotting!  the rand stuff should always go last.  fixed.